### PR TITLE
Update shortform submit options to auto-generate the post

### PR DESCRIPTION
**Pull request by jimrandomh**
~Currently doesn't work because submitting a shortform comment without a postId crashes (due to a preexisting backend bug that also affects `/shortform`.) After that's fixed the rest of this will be testable.~

### CODE REVIEW

**Comment by jimrandomh** [comment]
Based off PR: https://github.com/LessWrong2/Lesswrong2/pull/2159 . Known issues:

- [ ] When you submit a shortform post via the Recent Discussion widget, it doesn't refresh, so it looks like your comment has gone into the void (until you refresh).
- [ ] The Post to Shortform dialog is kind of confusing

**Comment by Raemon** [comment]
Also updated it to un-beta-gate the relevant features, since we're basically trying to go live now.

File: packages/lesswrong/components/common/Home2.jsx [hunk 1]
```diff
@@ -4,49 +4,25 @@ import React, { PureComponent } from 'react';
...
-              <SectionButton>
-                <AddBoxIcon />
-                New Shortform Post
-              </SectionButton>
-            </div>
-          </SectionTitle>
-          {showShortformFeed && <ShortformSubmitForm/>}
-          <RecentDiscussionThreadsList terms={{view: 'recentDiscussionThreadsList', limit:20}}/>
-        </SingleColumnSection>
+        <RecentDiscussionThreadsList terms={{view: 'recentDiscussionThreadsList', limit:20}}/>
```
Comment (jimrandomh) [inline packages/lesswrong/components/common/Home2.jsx]: @Raemon This had copy-paste damage; you have to also apply the same change in `HomeEA.jsx`.

File: packages/lesswrong/components/shortform/ShortformSubmitForm.jsx [hunk 2]
```diff
@@ -4,14 +4,18 @@ import { Components, registerComponent } from 'meteor/vulcan:core';
 
 const ShortformSubmitForm = ({currentUser, successCallback}) => {
   const { CommentsNewForm } = Components;
+  const shortformFeedId = currentUser?.shortformFeedId
+
   return (
     <CommentsNewForm
+      post={{_id:shortformFeedId}}
```
Comment (jimrandomh) [inline packages/lesswrong/components/shortform/ShortformSubmitForm.jsx]: I took that out deliberately. The API is that matching up to the shortform feed container is done server side.

File: packages/lesswrong/components/users/UsersMenu.jsx [hunk 3]
```diff
@@ -79,16 +79,16 @@ class UsersMenu extends PureComponent {
         >
             {showNewButtons &&
               <MenuItem onClick={()=>openDialog({componentName:"NewQuestionDialog"})}>
-                Ask Question [Beta]
+                Ask Question
```
Comment (jimrandomh) [inline packages/lesswrong/components/users/UsersMenu.jsx]: I'm not opposed to this change per se, but I don't like unrelated things sneaking into PRs like this.

**Pull request merged**